eng/pipelines/templates/jobs/analyze.yml

```diff
@@ -10,6 +10,7 @@ jobs:
   dependsOn: []
   steps:
   - checkout: self
+    fetchDepth: 0
```

Comment:
Another alternative is to switch to sparse checkout which should pull in the needed history as needed.

Reply:
I would in a larger repo.  This repo is tiny and should stay small for a long time